README.md

```diff
@@ -48,6 +48,12 @@ libgcrypt-devel libselinux-devel libxml2-devel libxslt-devel make openldap-devel
 pcre-devel perl-XML-Parser perl-XML-XPath perl-devel python-devel rpm-devel swig \
 bzip2-devel
 ```
+On Ubuntu 16.04 the command to install these package is
+```code
+sudo apt-get install -y autoconf automake libtool make libdbus-1-dev libdbus-glib-1-dev libcurl4-openssl-dev libgcrypt20-dev libselinux1-dev libxslt1-dev libgconf2-dev libacl1-dev libblkid-dev libcap-dev libxml2-dev libldap2-dev libpcre3-dev python-dev swig libxml-parser-perl libxml-xpath-perl libperl5.22 python-dev libbz2-dev librpm-dev swig
+
```

Comment:
This blank line in the code block below `apt-get install` seems distracting to me, does it have any purpose?

Reply:
Hm, not really. I thought there was a ``` to end the code block but the preview looked OK